### PR TITLE
Add display of population information

### LINKED ISSUE #25: simplifiy development setup by enabling CORS on geoserver
Geoserver appears to have an environment variable, [CORS_ENABLED](https://github.com/pacificclimate/scip/blob/dockerize/docker/docker-compose.yaml#L18) that would set it to share information. 

Currently, the SCIP setup is complicated by the use of a proxy to get around geoserver's CORS restrictions, so if we could update this variable in our development geoserver, setting up SCIP for development or deployment would be much easier!

### LINKED ISSUE #35: display salmon populations in selected area
Using the [new backend](https://github.com/pacificclimate/scip-backend), display information about the salmon populations (if any) found in the selected area.

### PR DESCRIPTION
**Pull request by corviday**
This PR incorporates data about the location of salmon populations from the SCIP backend into the front end and displays it as a table. 

It also removes geoserver from the system - the SCIP backend is replacing that functionality.

Demo can be seen [here](https://services.pacificclimate.org/dev/scip/app/), the changes are visible in the "Salmon Populations" tab, after you select a watershed from the dropdown menu under the map. 

resolves #35 
resolves #25

### CODE REVIEW

**Comment by corviday** [comment]
a UI question:

pink salmon subgroups are "odd" and "even" in the data files and in the database, but would we find it clearer if the website said "odd year" and "even year"?

**Review by rod-glover — COMMENTED** [review]
Great work. 

I see a lot of care with object naming, which is so important for maintenance. 

Code style is variable and kind of idiosyncratic; I think a pass through [ESLint](https://eslint.org/) would be a good idea. Many IDE's have an ESLint plugin that makes this easy. See climate-explorer-frontend for an example of use.

**Comment by rod-glover** [comment]
Argh, review not complete. Please wait.

File: src/components/PopulationTable/PopulationTable.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    ];
+    
+    //this is different than merely waiting for data to load.
+    //if we are waiting for an initial data load, populations 
+    //is undefined. If there are no salmon in an area, populations
+    //is an empty list.
+    function noSalmonHere()
+    {
+        return Array.isArray(populations) && populations.length===0;
+    }
```
Comment (rod-glover) [inline src/components/PopulationTable/PopulationTable.js:40]: Nice to pull out the meaning with the name!

This need not be a function though. You can just calculate a derived value, and simplify code:

```
const noSalmonHere = Array.isArray(populations) && populations.length===0;
```

File: package.json [hunk 2]
```diff
@@ -3,11 +3,19 @@
   "version": "0.1.0",
   "homepage": ".",
   "private": true,
+  "license": "GPL-3.0",  
+  "contributors": [
+    "James Hiebert <hiebert@uvic.ca>",
+    "Lee Zeman <lzeman@uvic.ca>",
```
Comment (rod-glover) [inline package.json:9]: Don't be so modest! You're the primary author, put yourself at the top.

**Review by rod-glover — CHANGES_REQUESTED** [review]
Definitely looking good. I have only one substantive change request:
- I think `useEffect` is being misused in one place.

Code style definitely needs some attention. That would be better in a separate PR. I was going to create an issue, but I see you already have one. It might be really helpful to you to look at ESLint integration into your IDE; I find it so.

**Comment by rod-glover** [comment]
I've put further UI feedback in the issue for that #34 . That way when/if others do the same it can all be in one place.

File: src/components/PopulationTable/PopulationTable.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+        function tableizePop(pop) {
+            return {
+                code: pop["conservation_unit"]["code"],
+                name: pop["conservation_unit"]["name"],
+                species: `${pop["common_name"]} ${pop.subgroup? "- " + pop.subgroup: ""}` 
+            }
+        }
+        setPopulationRows(_.map(populations[0], tableizePop));
+        }
+  }, [populations]);
```
[the remark this answers is not in the document]
Comment (corviday) [reply src/components/PopulationTable/PopulationTable.js:25]: It featured in all the sample code documentation for AgGridReact, the toolkit I used for the table, but I must have misunderstood the sample code.

**Review by rod-glover — APPROVED** [review]
LGTM!

**Review by eyvorchuk — APPROVED** [review]
Looks good to me overall! Would you be able to make the "Name" column bigger in the Salmon Populations table? The longer names get cut off.

**Comment by corviday** [comment]
Hey @eyvorchuk , that's a great suggestion! I did that.
tablespace

**Pull request merged**